cliquet_docs/api/resource.rst

```diff
@@ -876,7 +876,7 @@ In a response, ``permissions`` contains the current permissions of the record
 
 .. note::
 
-    When a record is created or modified, the current :term`user id`
+    When a record is created or modified, the current :term:`user id`
     **is always added** among the ``write`` principals.
 
-`Read more about leveraging resource permissions <resource-permissions>`.
+:ref:`Read more about leveraging resource permissions <permissions>`.
```

Comment:
Well, the link is broken right now anyhow (see the end of [this page](http://kinto.readthedocs.org/en/latest/api/1.x/cliquet/resource.html#notes-on-permissions-attribute)), so this commit doesn't make things any worse.

I tried it both ways (after disabling the re-copy part of `docs/conf.py` by hand), and it's a dead link either way, because both `resource-permissions` and `permissions` point to the `reference` directory of the Cliquet docs (to `resource.rst` and `permission.rst`, respectively), which Kinto doesn't copy.

I think that the way Kinto copies Cliquet docs is confusing and strange and I think it needs to be rethought, but that's outside the scope of this PR.